types/setting.go

```diff
@@ -255,6 +258,18 @@ var (
 		Default:     "300",
 	}
 
+	SettingDefinitionFailedBackupTTL = SettingDefinition{
+		DisplayName: "Failed Backup Time to Live",
+		Description: "In seconds. This setting determines how long Longhorn will keep the backup resource that was failed. Set to 0 to disable the auto-deletion.\n" +
+			"The failed backup will be removed in **BackupstorePollInterval** (default: 300 seconds.) " +
```

Comment:
Suggest to `Failed backups will be checked and cleaned up during backupstore polling which is controlled by **BackupstorePollInterval** setting.`